snapcraft/cli/_errors.py

```diff
@@ -29,8 +30,21 @@
 except ImportError:
     RavenClient = None
 
-_MSG_INTERNAL_ERROR = 'Sorry, Snapcraft had an internal error.'
-_MSG_SEND_TO_SENTRY = 'To help us improve, would you like to send error data?'
+# TODO:
+# - annotate the part and lifecycle step in the message
+# - add link to privacy policy
+# - add Always option
+_MSG_SEND_TO_SENTRY_TRACEBACK = dedent("""\
+    Sorry, Snapcraft ran into an error when trying to running through its
+    lifecycle that generated the following traceback:""")
```

Comment:
Since we don't have the lifecycle step etc. anyway, might as well keep it more brief, perhaps:

"Sorry, Snapcraft ran into an error that resulted in the following traceback:"

Reply:
ok, please comment on the doc as well

Reply:
No the doc is fine, this is really relating to the TODO, i.e. we need to know the lifecycle step before discussing it is actually useful. Once we have that, the doc works.